svc/api/routes/v2_apis_list_keys/handler.go

```
@@ -309,7 +309,7 @@ func (h *Handler) buildKeyResponseData(keyData *db.KeyData, plaintext string) op
 	if keyData.Key.RemainingRequests.Valid {
 		response.Credits = &openapi.KeyCreditsData{
 			Refill:    nil,
-			Remaining: nullable.NewNullableWithValue(int64(keyData.Key.RemainingRequests.Int32)),
+			Remaining: nullable.NewNullableWithValue(int64(keyData.Key.RemainingRequests.Int64)),
```

golangci-lint (2.11.4)

[error] unnecessary conversion

(unconvert)